pkg/controller/rotation/metrics.go

```diff
@@ -59,14 +59,14 @@ func init() {
 			Aggregation: view.Count(),
 		},
 		{
-			Name:        metricPrefix + "/token_success",
```

Comment:
Had to rename these so we can use the same MQL query for .success (they are new metrics I just created in the last PR, so no prod impact).